met-web/src/components/engagement/new/create/index.tsx

```diff
@@ -123,8 +123,9 @@ const EngagementCreationWizard = () => {
 
     const handleLanguageEntries = (languages: Language[]) => {
         return languages.map((language) => {
-            const languageWithDirection =
-                true === language.right_to_left ? language.name.split('').reverse().join('') : language.name;
+            const languageWithDirection = language.right_to_left
+                ? language.name.split('').reverse().join('')
+                : language.name;
             return (
                 <>
                     <span>
```

Comment:
```suggestion
            return (
                <>
                    <span style={{ direction: language.right_to_left ? 'rtl' : 'ltr' }}>

Reply:
Maybe I don't have the full context here but I don't believe we should implementing any logic to handle RTL languages, whether it's reversing a string or adding styling. If this is a simple field for selecting a language then I don't think engagement editors will expect the name of the language itself to show up reversed in some cases. 

The ticket doesn't make mention of RTL so I would check with the PO on this, or just omit language direction handling for now